uxdocs/04. Componenten/badge.md

```diff
@@ -0,0 +1,179 @@
+
+### Dot indicator
+
+A dot indicator is a badge without content. The context of a dot indicator is defined by its parent element.
+
+### Text badge
+
+A text badge can be used to signify a status or category.
+
+### Status light
+
+A status light is used to display a status. A status light badge should have a descriptive text label.
+
+## Alternatives and related components
+
+## Anatomy
+
+- Container with padding
+- Standard right and bottommargin
+- Text (except for the Dot badge)
+- Text label (for the status light)
+- Optional icon
+
+## Design tokens
+
+- badge-corner-radius // Border radius
+- badge-padding-inline // Left and right padding
+- badge-padding-block // Top and bottom padding
+- badge-background-color // Background color
+- badge-foreground-color // Badge text foreground color
+- badge-font-family // Badge text font-family
+- badge-font-size // Badge text font size
+- badge-font-weight // Badge text font weight
+
+### Counter badge
+
+### Dot indicator
+
+### Text badge
+
+- badge-text-transform // Sentence case or uppercase
+
+### Status light success
+
+- feedback-fill-safe-background-color // Success notification background color
+- feedback-fill-safe-color // Success notification foreground (icon) color
+
+### Status light warning
+
+- feedback-fill-warning-background-color // Warning notification background color
+- feedback-fill-warning-color // Warning notification foreground (icon) color
+
+### Status light error
+
+- feedback-fill-danger-background-color // Error notification background color
+- feedback-fill-danger-color // Error notification foreground (icon) color
+
+## Interactive states
+
+If a badge is interactive, for instance when it is being used as category navigation, make sure that all interactive states are properly designed.
+
+### Focus
+
+### Hover
+
+### Active
+
+## Design properties
+
+### Positioning
+
+In case multiple badges are placed next or above each other make sure there is enough spacing between each badge.
+
+### Typography
+
+If a badge displays a status, text can be set in all-caps.
+
+### Inheritance of text size
+
+A badge can inherit the text-size of its parent element, for example when a badge is used inside a heading.
+
+## Examples
+
+## Accessibility
+
+Make sure there is enough contrast between the fore- and background colors.
+
+Do not rely on color alone to indicate a status, make sure people suffering from forms of color blindness are able to see a difference in status by adding texture. Use for instance a filled versus an outlined badge to indicate differences.
+
+When a badge indicates an update
+
+In case a badge displays an update, make sure that people relying on screen readers are made aware of such updates as well.
+
+## Content guidelines
+
+Try to minimize text inside a badge to a single word. In extreme cases up to a maximum of three words should be used.
+
+## Best practices
+
+### Dos
```

Comment:
For the status-badge I think you just cannot have multiple status-badges in a row, and there might even more specific guidelines on when to place them :)